app/forms/idv/state_id_form.rb

```diff
@@ -23,6 +23,9 @@ def submit(params)
       cleaned_errors = errors.deep_dup
```

Comment:
@svalexander I realized I made a mistake when originally working on this, and that's why `FakeAnalytics` didn't catch the issue here. This change will fix the issue.

```suggestion
      cleaned_errors = errors.dup
```

I'd appreciate if you make the same change to `address_form.rb` as well.

@aduth FYI I think this is part of the issue here, i.e. `ActiveModel::Error` supports `dup` for cloning, but not `deep_dup`. There may also be issues when trying to print out the contents of these error objects directly.